hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/XceiverClientFactory.java

```diff
@@ -42,4 +42,6 @@ XceiverClientSpi acquireClient(Pipeline pipeline, boolean topologyAware)
   void releaseClient(XceiverClientSpi xceiverClient, boolean invalidateClient,
                      boolean topologyAware);
 
+  XceiverClientSpi acquireClientUncached(Pipeline pipeline, boolean topologyAware)
+      throws Exception;
```

Comment:
Instead of adding a new method to the `XceiverClientFactory` interface, how about creating a new, non-caching implementation as parent class of `XceiverClientManager`?

Reply:
see https://github.com/apache/ozone/commit/ad95eb87c6d57f890480e8332ab2584058903ad9